### PR TITLE
feat(application): add ui to display user list

### LINKED ISSUE #106: display list of users logged into application
currently, application creators cannot see the statistics of their application or view reports.

Do the following.
- [ ] open new page by clicking on the "details" button on the application card.
- [ ] Display names of people who have logged into the application.

## Method of submission, one PR for each of the following.
- [ ] Forntend Interface to list details
- [ ] Backend logic to list user names.

### PR DESCRIPTION
**Pull request by MASTERCHIEF011**
Fixes #106 
Fixes Frontend part.
Screen Shot 
image

### CODE REVIEW

File: views/dashboard/details.hbs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                    </div>
+                </div>
+            </div>
+    </section>
+
+
+
+</body>
+<script src="../assets/js/util.js"></script>
+
```
Comment (YashKumarVerma) [inline views/dashboard/details.hbs:133]: no need to add util here

File: views/dashboard/dev.hbs [hunk 2]
```diff
@@ -159,10 +159,12 @@
...
-                        <button
-                            class="flex items-center px-4 py-2 mx-auto mt-auto font-semibold text-black duration-500 ease-in-out transform bg-white border rounded-lg shadow-xl transittion hover:text-white hover:bg-black hover:border-black focus:shadow-outline focus:outline-none">
-                            View Details
-                        </button>
+                        <form action="/dashboard/dev/details/{{_id}}" method="GET">
+                            <button
+                                class="flex items-center px-4 py-2 mx-auto mt-auto font-semibold text-black duration-500 ease-in-out transform bg-white border rounded-lg shadow-xl transittion hover:text-white hover:bg-black hover:border-black focus:shadow-outline focus:outline-none">
+                                View Details
+                            </button>
+                        </form>
```
Comment (YashKumarVerma) [inline views/dashboard/dev.hbs:167]: no need to create form, use simple anchor tag and take to new link

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: views/dashboard/details.hbs (hunk 1 above)
Comment (MASTERCHIEF011) [reply views/dashboard/details.hbs:133]: OKAY

File: views/dashboard/dev.hbs (hunk 2 above)
Comment (MASTERCHIEF011) [reply views/dashboard/dev.hbs:167]: But this link is sending the id of the app which i dont think can be achieved my an anchor tag

**Comment by YashKumarVerma** [comment]
add screenshots of UI

**Review by YashKumarVerma — APPROVED** [review]

**Comment by YashKumarVerma** [comment]
thanks !

**Pull request merged**